corehq/apps/app_manager/views/apps.py

```diff
@@ -809,16 +814,13 @@ def parse_sync_interval(interval):
         ('mobile_ucr_restore_version', None),
         ('location_fixture_restore', None),
     )
-    linked_app_attrs = [
-        'target_commcare_flavor',
-    ]
     for attribute, transformation in easy_attrs:
         if should_edit(attribute):
             value = hq_settings[attribute]
             if transformation:
                 value = transformation(value)
             setattr(app, attribute, value)
-            if hasattr(app, 'linked_app_attrs') and attribute in linked_app_attrs:
+            if app.get_doc_type() == 'LinkedApplication' and attribute in app.SUPPORTED_SETTINGS:
```

Comment:
may be worth having a helper like https://github.com/dimagi/commcare-hq/blob/7b91e11db1f71cd0ccf2ed5f783a9c46c65356d0/corehq/apps/app_manager/models.py#L4201-L4202 at some point for linked apps

Reply:
yeah, that makes sense. not going to include in this PR.